src/java.desktop/share/classes/java/awt/TextArea.java

```diff
@@ -606,6 +606,15 @@ public Dimension minimumSize() {
         }
     }
 
+    @Override
+    public synchronized void setEditable(boolean b) {
```

Comment:
`setEditable` method is overridden in both `TextArea` and `TextField` class and method implementation is same. Since both of them are  inherited from `TextComponent` class, could we move the code to `TextComponent's setEditable` method and remove them from respective classes?

Reply:
This is a good suggestion.